site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md

```diff
@@ -581,6 +581,171 @@ Click on the buttons in the top menu to toggle between the two modes (_#1. in th
+
+##### Code
+
+Review the [source code](https://github.com/privacysandbox/noise-lab/blob/main/public/index.html#L43)
+for APE calculation.
+
+
+#### RMSPE_T  (root-mean-square percentage error with a threshold) {: #rmspe}
+
+RMSPE_T (root-mean-square percentage error with a threshold) is another measure for noise.
+
+##### How to interpret RMSPE_T
+
+Lower RMSPE_T values mean better signal-to-noise ratios. 
+
+RMSPE_T is typically higher than the average percentage error. For example, if a noise ratio that's acceptable for your use case is 20%, and RMSPE_T is 20%, you can be confident that noise levels fall into your acceptable range.
+
+{% Aside %}
+What does it mean if RMSPE_T is above 100%?
+The interpretation of  RMSPE_T over 100% is similar to that of an APE over 100% LINK.
+
+RMSPE (Root Mean Square Error) is a common metricused by data scientists. 
+RMSPE_T is a variation of RMSPE. RMSPE_T differs from RMSPE in two ways:
+
+1. It uses a percentage error similar to noise-to-signal rate, like noise/true-value. 
+2. It assumes the existence of minimal signal: the percentage looks like noise/max(true, T). 
+
+{% endAside %}
+
+##### Formula
+
+For a given summary report, RMSPE_T is calculated as follows:
+
+<figure>
+{% Img src="image/URLGRmk9LjR39BLvmeGDZFZkz3p2/zNE4v707JUoUS8keFVsz.png", alt="Formula for RMSPE_T", width="377", height="59" %}
+<figcaption style="text-align:left">
+  The equation for RMSPE_T. Absolute values are required, as noise can be negative.
+</figcaption>
+</figure>
+
+In Noise Lab, this is then multiplied by 100 to give a percentage.
+
+##### Pros and cons
+
+RMSPE_T is a bit more complex to grasp than APE. However, it has a few advantages that make it in some cases more suitable than APE for analyzing noise in summary reports:
+
+* **It's more stable.** *"T"* is a threshold: it's used to give less weight, in the RMSPE_T calculation, to buckets that have less conversions and are therefore more sensitive to noise due to their small size. With T, the metric does not spike on buckets with few conversions. If T is equal to 5, a noise value as small as 1 on a bucket with 0 conversion will not be displayed as way over 100%. Instead, it will be capped at 20%, which is equivalent to 1/5, as T is equal to 5. By giving less weight to buckets that are more sensitive to noise due to their small size, this metric is more stable — and therefore makes it easier to compare two simulations.
+* **It allows for easy aggregation.** Knowing RMSPE_t of multiple buckets, together with their true counts, allows one to compute the RMSPSE_t of their sum via a simple formula. This also allows one to easily optimize for RMSPE_t of these combined values. (While aggregation is possible for APE, the formula is quite complicated since it involves the absolute value of sum of Laplace noises, which makes it hard to optimize.) 
```

Comment:
```suggestion
* **RMSPE_T is more stable.** "T" is a threshold. "T" is used to give less weight in the RMSPE_T calculation to buckets that have less conversions and are therefore more sensitive to noise due to their small size. With T, the metric does not spike on buckets with few conversions. If T is equal to 5, a noise value as small as 1 on a bucket with 0 conversions will not be displayed as way over 100%. Instead, it will be capped at 20%, which is equivalent to 1/5, as T is equal to 5. By giving less weight to smaller buckets which are therefore more sensitive to noise, this metric is more stable, and therefore makes it easier to compare two simulations.
* **RMSPE_T allows for easy aggregation.** Knowing the RMSPE_T of multiple buckets, together with their true counts, allows you to compute the RMSPE_T of their sum. This also allows you to optimize for RMSPE_T for these combined values.

While aggregation is possible for APE, the formula is quite complicated as it involves the absolute value of sum of Laplace noises. This makes APE harder to optimize.
```